commands/Factorio/amilinked.js

```diff
@@ -0,0 +1,32 @@
+const { MessageEmbed } = require("discord.js");
+const Command = require("../../base/Command.js");
+
+class Linkme extends Command {
+  constructor(client) {
+    super(client, {
+      name: "amilinked",
+      description: "Check linking status of a user",
```

Comment:
This description could read better:

`Check if a user is linked with factorio` perhaps?